compiler/rustc_target/src/spec/targets/aarch64_apple_ios.rs

```diff
@@ -18,6 +18,7 @@ pub fn target() -> Target {
         options: TargetOptions {
             features: "+neon,+fp-armv8,+apple-a7".into(),
             max_atomic_width: Some(128),
+            stack_probes: StackProbeType::Inline,
```

Comment:
Do we need to gate use of `StackProbeType::Inline` on LLVM 18 or newer? Is this just ignored for unsupported targets on earlier versions of LLVM?

Reply:
It's ignored -- or rather goes unchecked and unnoticed. The other times we've concerned ourselves about the version is when we know there are bugs in the implementation.